rep-0003.rst

```diff
@@ -281,6 +281,68 @@ Build System Support:
 
 - Same as Indigo
 
+Melodic Morenia (May 2018 - May 2023)
+--------------------------------------
+Required Support for:
+
+- Ubuntu Artful (17.10)
+- Ubuntu B (18.04)
+
+Recommended Support for:
+
+- Debian Stretch
+- Fedora 28
+
+Architectures Supported:
+
+- amd64
+- arm32
+- arm64
+
+Targeted Languages:
+
+- C++11
+- Python 2.7
+
+  - Python >= 3.5 not required, but testing against it is recommended
+
+- Lisp SBCL 1.3.14
+
+Requirements:
+
+
++---------+---------------+----------------+----------------+-----------+
+| Package | Ubuntu Artful | Ubuntu B       | Debian Stretch | Fedora 28 |
++=========+===============+================+================+===========+
+|         |       Required Support         |    Recommended support     |
++---------+---------------+----------------+----------------+-----------+
+| Boost   |     1.62      |     1.62!      |    1.62        |   1.64!   |
++---------+---------------+----------------+----------------+-----------+
+| CMake   |     3.9.1     |     3.9.1!     |    3.7.2       |   3.9.3!  |
++---------+---------------+----------------+----------------+-----------+
+| Gazebo  |     7.5.0     |     7.5.0!     |    7.3.1       |   8.1.1!  |
```

Comment:
@j-rivero do you think gazebo 8 will make it into Ubuntu 18.04 ? If not, will gazebo7 be maintained until 2023 ?

Reply:
@mikaelarguedas do you mean to ask about Gazebo 9.0? 8.0 isn't an LTS afaik

Reply:
Indeed I meant Gazebo 9 :confounded:, thanks @IanTheEngineer for catching it. I'll edit the previous comment accordingly

Reply:
Gazebo 9 has a big number of useful functionality compared to 7. It'd be really sad to stay with 7 in Melodic. But I know it depends on the development status and decision about Gazebo 9 target platforms (which I couldn't find).

Reply:
@j-rivero Gentle ping, any thoughts on the versions of Gazebo here, and what we can expect will be supported for the life of Melodic?

Reply:
We are working for getting gazebo9 into Ubuntu 18.04 but I can not say yes or no at this moment since there is a good bunch of packages that need to go through the Debian review process, which takes an unpredictable amount of time. The different supporting periods are listed in the [gazebo webpage](http://gazebosim.org/#status).